build.yaml

```diff
@@ -2,7 +2,7 @@ builders:
   less_dart:
     import: 'package:less_dart/less_builder.dart'
     builder_factories: ['lessBuilder']
-    auto_apply: none
+    auto_apply: dependents
```

Comment:
This can simplify builder configuration - it will be automatically enabled for all packages, which have less_dart in dependencies